docs: reformat the documentation (#396)
ansys/ansys-sphinx-theme: doc/source/examples/sphinx-gallery/README.rst

:orphan:

> Sphinx gallery

Reviewer: ```suggestion
Sphinx-Gallery
```

==============

> This is an example of a gallery generated with `Sphinx gallery`` and its
> rendering with ansys sphinx theme.

Reviewer: ```suggestion
This example shows how the Ansys Sphinx Theme renders examples
using Sphinx Gallery. 
```
